examples/twentyseven-pt-stencil-solver/twentyseven-pt-stencil-solver.cpp

```diff
@@ -93,10 +93,12 @@ constexpr double default_delta = -1.0 / 24.0;
 
 // Creates a stencil matrix in CSR format for the given number of discretization
 // points.
-void generate_stencil_matrix(int dp, int *row_ptrs, int *col_idxs,
-                             double *values, double *coefs)
+template <typename ValueType, typename IndexType>
```

Comment:
In this and also the 9pt stencil example, line 82->85 we have all the default values declared as double. I'm not sure if in float that that creates issues due to the computed precision?

Reply:
the coefs are stored as `ValueType` in line 355->374. So, I think it should be fine.